packages/manager/modules/core/src/redirection/redirection.constants.js

```diff
@@ -16,6 +17,7 @@ const URI = {
 export default {
   EU: {
     billing: `${managerRoot.EU}/manager/dedicated/${URI.billing}`,
+    catalog: `${managerRoot.EU}/manager/${URI.catalog}`,
```

Comment:
/manager/dedicated ?

Reply:
maybe we should add something like
```
managerUri : {
  dedicated : {
    EU: '',
    CA: '',
  }
}
```
to avoid making those mistakes

Reply:
No `catalog` is in `manager-hub` so it is `manager/`